src/Button/GreenButton.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+import Button from './Button'
+
+const GreenButton = styled(Button)`
```

Comment:
👍

Reply:
Not to open a can of worms, but it might be worth sharing this pattern with your team and getting a sense of what they think of the naming convention here